shared.js

```diff
@@ -0,0 +1,45 @@
+/*
+ * This config is shared between the default and react configs
+ */
+module.exports = {
+  plugins: [
+    "jest",
+    "jsdoc",
+    "sort-keys-fix",
+    "sort-destructure-keys",
+    "node",
+    "@typescript-eslint",
+  ],
+  rules: {
+    "brace-style": ["error", "stroustrup"],
+    "comma-dangle": ["error", "always-multiline"],
+    curly: ["error", "multi-line", "consistent"],
+    "no-unused-vars": "off",
+    "@typescript-eslint/no-unused-vars": [
+      "error",
+      { argsIgnorePattern: "^_", ignoreRestSiblings: false },
+    ],
+    "no-shadow": "off",
+    "@typescript-eslint/no-shadow": ["error"],
+    "node/no-deprecated-api": "warn",
+    semi: "off",
+    "@typescript-eslint/semi": ["error", "never"],
+    "sort-destructure-keys/sort-destructure-keys": "error",
+    "sort-keys-fix/sort-keys-fix": ["error", "asc", { natural: true }],
+    "import/prefer-default-export": "off",
+    "prefer-destructuring": "off",
+    "import/no-cycle": "off",
```

Comment:
Why are we turning these off?

Reply:
Because Leap turns it off. But you're right, I'll remove it from here and keep it as a Leap specific override

Reply:
Where does Leap turn off `import/prefer-default-export`? We're turning off `import/no-cycle` purely because there were too many fixes at the time, but there is a ticket to restore this rule, same with `prefer-destructuring`.

Reply:
Looks like it was turned off when I copied them over but has been removed since. I've now removed it.